artifacts/definitions/Linux/Sys/Trash.yaml

```diff
@@ -0,0 +1,24 @@
+name: Linux.Sys.Trash
+description: Find files in trash bin for all users
+author: Whitney Champion (@shortxstack)
+parameters:
+  - name: trashFiles
+    default: '.local/share/Trash/**/*'
+    description: Glob of trash bin files relative to a user's home directory.
+
+sources:
+  - precondition: |
+      SELECT OS From info() where OS = 'linux'
+
+    query: |
+      LET trash = SELECT * from foreach(
+          row={
+             SELECT Uid, User, Homedir from Artifact.Linux.Sys.Users()
+          },
+          query={
+             SELECT FullPath, Mtime, Ctime, User, Uid
```

Comment:
Could you switch this to OSPath? Is it worth putting a Size field too?

Reply:
fixed/added :)